zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs

```diff
@@ -11,7 +11,7 @@ use std::{env::var, vec};
 
 pub(crate) fn get_num_rows_per_round() -> usize {
     var("KECCAK_ROWS")
-        .unwrap_or_else(|_| "5".to_string())
+        .unwrap_or_else(|_| "12".to_string())
```

Comment:
@davidnevadoc why this increase?

Reply:
as we discussed with Han previously，circuit being wider is friendly for super circuit, But the more columns the more  proving cost of aggregation circuit. So we did not choose minimum possible value But a bit larger

Reply:
Ok